doc/integrator/make.rst

```diff
@@ -84,7 +84,6 @@ The following variables may be set in the makefiles:
 * ``CGXP_INTERFACES``: List of CGXP interfaces, default is empty.
 * ``NGEO_INTERFACES``: List of ngeo interfaces, default is ``mobile desktop``.
 * ``PRINT``: Mapfish print is enable, default to ``TRUE``.
```

Comment:
enable -> enabled
default to -> default is

Reply:
Thanks, updated :-)